fe/fe-core/src/main/java/com/starrocks/statistic/StatisticExecutor.java

```diff
@@ -303,6 +305,22 @@ public List<TStatisticData> executeStatisticDQL(ConnectContext context, String s
     private List<TResultBatch> executeDQL(ConnectContext context, String sql) {
         StatementBase parsedStmt = SqlParser.parseFirstStatement(sql, context.getSessionVariable().getSqlMode());
         ExecPlan execPlan = StatementPlanner.plan(parsedStmt, context, TResultSinkType.STATISTIC);
+
+        List<PhysicalOlapScanOperator> pos = Utils.extractPhysicalOlapScanOperator(execPlan.getPhysicalPlan());
+        StringBuilder stringBuilder = new StringBuilder();
+        for (PhysicalOlapScanOperator scan : pos) {
+            Table table = scan.getTable();
+            stringBuilder.append("[table: ").append(table.getName()).append(", ");
+            for (Partition partition : table.getPartitions()) {
+                stringBuilder.append(partition.getName()).append(": ");
+                stringBuilder.append(partition.getCommittedVersion()).append(", ");
+                stringBuilder.append(partition.getVisibleVersion()).append(", ");
+                stringBuilder.append(partition.getVisibleVersionTime());
+            }
+            stringBuilder.append("]");
+        }
+        LOG.debug("statistics DQL plan tables: " + stringBuilder);
```

Comment:
We'd better move the build stringBuilder() process in `if (enableDebug)` block.

Reply:
OK